code/Controllers/SilverStripeNavigatorItem_LiveLink.php

```diff
@@ -58,11 +58,20 @@ public function canView($member = null)
             $record->hasExtension(Versioned::class)
             && $record->hasStages()
             && $this->getLivePage()
-            // Don't follow redirects in preview, they break the CMS editing form
-            && !($this->record instanceof RedirectorPage)
+            && $this->showLiveLink()
         );
     }
 
+    public function showLiveLink()
+    {
+        try {
+            return $this->record->config()->get('show_live_link');
```

Comment:
Maybe cast this to `(bool)`, in case it's `null` or something odd?

Reply:
For more flexibility with configuration you could just use `Config::inst()->get(get_class($this->record), 'show_live_link')` instead. Then you can remove the `try...catch`.

Reply:
Updated 👍